src/Engine/Renderer/RenderObject/RenderObject.hpp

```diff
@@ -27,33 +27,33 @@ namespace Ra {
 namespace Engine {
 
 // Radium V2 : need a bit of cleanup !
```

Comment:
This comment is meaningless: many changes have been done, so if something is still missing, please state it explicitly.